Potser l'Eric té raó

BinarySandia04 opened the pull request:
Eric t'he fet el codi millor, siusplau accepta'l no sé si funciona pero hauria de fer-ho

BinarySandia04 committed:
L'eric no sap programar

BinarySandia04 commented:
**BTW CAL ACLARIR UNA COSA**

La funció `def isInt(s)` no converteix les strings en int, només comprova si es pot transformar, la he liat a l'hora de fer el comentari, i crec que cal que ho tingui en compte ja que la conducta del codi diu que els comentaris han de ser clars, concissos i entenedors i per tant, cal destacar aquest petit error humà.

royalmo reviewed (changes requested):
Doncs si has vist que falta convertir a int, fes el canvi en un commit (es posara automaticament dins la PR si ho fas a la mateixa branch)

royalmo committed:
Fixed Aran's work

Fixed Aran's work when he said I didn't know to program but he did it worse than me xD

royalmo commented on triplebot.py:
```diff
@@ -353,6 +361,72 @@ async def on_message(self, message):
+                    while vc.is_playing():
+                        await asyncio.sleep(.1)
+
+                    code = code*timesInt
```
Crak fiera mastodonte has de multiplicar-ho com a mínim, pero aixo et pot petar igualment crak fiera titan furia.

Que passa si poso `!code JKDSFH -3`

royalmo commented on triplebot.py:
```diff
@@ -353,6 +361,72 @@ async def on_message(self, message):
                 # await message.delete()
                 return
+        if len(content.split())==3:
+            laputaeric = content.split()
```
Hauries pogut agafar algo mes family friendly

royalmo reviewed (changes requested):
Que passa si poso `!code JKDSFH -3`?

royalmo committed:
Fixed Aran's work... again

royalmo reviewed (approved):
Ja t'ho he arreglat pq veig que et costa una miqueta xD anda en 10 mins ho tens al bot

BinarySandia04 commented:
Wow Eric no m esperaba q possessis un nombre negatiu per multiplicar un string, no m havia adonat compte d aquest cas (igualment t ho vaig fer a cegues pq ni el vaig executar xd)
